pkg/infrastructure/openstack/clusterapi/clusterapi.go

```diff
@@ -54,12 +54,14 @@ func (p Provider) PreProvision(ctx context.Context, in clusterapi.PreProvisionIn
 		return fmt.Errorf("failed to tag VIP ports: %w", err)
 	}
 
-	// upload the corresponding image to Glance if rhcosImage contains a
-	// URL. If rhcosImage contains a name, then that points to an existing
-	// Glance image.
-	if imageName, isURL := rhcos.GenerateOpenStackImageName(rhcosImage, infraID); isURL {
-		if err := preprovision.UploadBaseImage(ctx, installConfig.Config.Platform.OpenStack.Cloud, rhcosImage, imageName, infraID, installConfig.Config.Platform.OpenStack.ClusterOSImageProperties); err != nil {
-			return fmt.Errorf("failed to upload the RHCOS base image: %w", err)
+	if installConfig.Config.Platform.Name() == openstack.Name {
```

Comment:
I'd rather have a check that excludes platforms that should be left out than the opposite:
```suggestion
	if installConfig.Config.Platform.Name() != powervc.Name {
```

Reply:
Changed